core/src/mock_query.rs

```diff
@@ -55,4 +55,11 @@ impl MongoStatement for MongoQuery {
     fn get_resultset_metadata(&self) -> &Vec<MongoColMetadata> {
         &self.resultset_metadata
     }
+
+    // Close the cursor by setting the current value and cursor to None.
```

Comment:
Decided to implement this for `MockQuery` as well so I could use this type in the unit tests -- turned out to be easier that way.